doctr/datasets/funsd.py

```diff
@@ -91,7 +91,9 @@ def __init__(
                 crops = crop_bboxes_from_image(img_path=os.path.join(tmp_root, img_path),
                                                geoms=np.asarray(box_targets, dtype=np_dtype))
                 for crop, label in zip(crops, list(text_targets)):
-                    self.data.append((crop, dict(labels=[label])))
+                    # filter labels with non utf-8 characters and whitespaces
+                    if not any(char in label for char in [' ', '☑', '☐', '\uf703', '\uf702']):
```

Comment:
don't we have a better way to filter those out? 
Perhaps encoding  decoding them through utf8 to check whether we get the same thing back?